mysql-test/suite/rpl/r/rpl_show_slave_hosts.result

```diff
@@ -8,13 +8,363 @@ START SLAVE IO_THREAD;
+Variable_name	Value
+rpl_semi_sync_slave_enabled	ON
+connection master;
+show status like 'Rpl_semi_sync_master_clients';
+Variable_name	Value
+Rpl_semi_sync_master_clients	2
+SHOW REPLICA HOSTS;
+Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack
+3	slave2	SLAVE_PORT	1	0-1-2	
+2	localhost	SLAVE_PORT	1	0-1-2	0-1-2
+# Test 6: Create new transactions on primary
+# Note:   New transactions on primary are sent to replicas
+#         and connected replicas (server_id=2 and server_id=3)
+#         should sync to the GTID state on primary.
+# -------------------------------------------
+create table t2(t2 int);
+insert into t2 values (2);
+show status like 'Rpl_semi_sync_master_no_tx';
+Variable_name	Value
+Rpl_semi_sync_master_no_tx	0
+show status like 'Rpl_semi_sync_master_yes_tx';
+Variable_name	Value
+Rpl_semi_sync_master_yes_tx	4
+SHOW REPLICA HOSTS;
+Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack
+3	slave2	SLAVE_PORT	1	0-1-4	0-1-4
+2	localhost	SLAVE_PORT	1	0-1-4	0-1-4
+show status like "Slaves_connected";
+Variable_name	Value
+Slaves_connected	2
+# Test 7: Simulate stop of the IO thread of replica
+# Note:   Expecting the result of show statement will not be visible
+# -------------------------------------------
+connection slave;
+show status like 'rpl_semi_sync_slave%';
+Variable_name	Value
+Rpl_semi_sync_slave_send_ack	4
+Rpl_semi_sync_slave_status	ON
+show variables like 'rpl_semi_sync_slave_enabled';
+Variable_name	Value
+rpl_semi_sync_slave_enabled	ON
+STOP SLAVE IO_THREAD;
+include/wait_for_slave_io_to_stop.inc
+connection master;
+show status like "Slaves_connected";
+Variable_name	Value
+Slaves_connected	1
+show status like 'Rpl_semi_sync_master_clients';
+Variable_name	Value
+Rpl_semi_sync_master_clients	1
+show status like 'rpl_semi_sync_slave%';
+Variable_name	Value
+Rpl_semi_sync_slave_send_ack	0
+Rpl_semi_sync_slave_status	OFF
+show variables like 'rpl_semi_sync_slave_enabled';
+Variable_name	Value
+rpl_semi_sync_slave_enabled	OFF
+SHOW REPLICA HOSTS;
+Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack
+3	slave2	SLAVE_PORT	1	0-1-4	0-1-4
+INSERT INTO t VALUES (10);
+show status like 'Rpl_semi_sync_master_yes_tx';
+Variable_name	Value
+Rpl_semi_sync_master_yes_tx	5
+SHOW REPLICA HOSTS;
+Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack
+3	slave2	SLAVE_PORT	1	0-1-5	0-1-5
+connection slave;
+START SLAVE IO_THREAD;
+include/wait_for_slave_io_to_start.inc
+show status like 'rpl_semi_sync_slave%';
+Variable_name	Value
+Rpl_semi_sync_slave_send_ack	0
+Rpl_semi_sync_slave_status	ON
+show variables like 'rpl_semi_sync_slave_enabled';
+Variable_name	Value
+rpl_semi_sync_slave_enabled	ON
+connection master;
+show status like "Slaves_connected";
+Variable_name	Value
+Slaves_connected	2
+show status like 'Rpl_semi_sync_master_clients';
+Variable_name	Value
+Rpl_semi_sync_master_clients	1
+SHOW REPLICA HOSTS;
+Server_id	Host	Port	Master_id	Gtid_State_Sent	Gtid_State_Ack
+2	localhost	SLAVE_PORT	1		
```

Comment:
This seems time-dependent, where sometimes `Gtid_State_Sent` could have `0-1-5`, and sometimes not, so I would suggest only keeping the `SHOW REPLICA HOSTS` after syncing the replica with the master